apps/dashboard/src/data/projects/selectors.js

```diff
@@ -21,3 +21,6 @@ export const isProjectPublic = ( state ) =>
 
 export const isProjectSaved = ( state ) =>
 	get( state, [ 'projects', 'isSaved' ], false );
+
+export const isEditDisabled = ( state ) =>
```

Comment:
Disabling the editing for a few seconds when we still don't have a saved project helps to avoid parallel project creation requests.

Reply:
I'm just curious... couldn't `isProjectSaving` be used to potentially achieve the same result?  
As in, we should send save requests at all if _any_ project is being saved. Regardless if it has an ID or not.

To give more context, you can see there's a bit of inconsistency in project state - some of it requires a project ID while other functions just have a 'global state' like the case here.  
I'd argue the latter is really better considered 'editor' state and would love to clean it up a little once we get the first iteration out of the door.

Reply:
The difference here is that `isEditDisabled` only returns true for new projects, without an Id.
The main reason for this issue was two parallel save requests (without a projectId) being sent to the backend and creating two new projects.
I couldn't think of a way to avoid this behavior without blocking those two requests from happening.
Once we get a projectId, we don't need to care about the parallel requests because the project will be patched every time.

Reply:
I see. Alright, let's keep it the way it is. Thanks for the explanation.